ops/op2/signature.rs

```diff
@@ -138,13 +138,14 @@ impl ToTokens for V8Arg {
   }
 }
 
-#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub enum Special {
   HandleScope,
   OpState,
   JsRuntimeState,
   FastApiCallbackOptions,
   Isolate,
+  CppGcResource(syn::Path),
```

Comment:
Use String here -- that's the current pattern for embedded types like this

Reply:
I don't this its good to use Strings. Eg: `quote!`-ing a `String` here becomes:

```rust
unwrap_cpp_object::<"std::fs::File">();
```

`parse_ident` wouldn't work because `std::fs::File` is not a ident. I'd have to reparse the String to a syn::Path during codegen.

Reply:
Yeah -- we currently re-parse those all through codegen in the other places as well. It's a tradeoff between keeping the Arg interface somewhat uncoupled from syn types.

It keeps the layers a little less tightly coupled. Let's use String for now -- if we decide to use syn types throughout the stack we should fix it for all the types.

Reply:
Done